scripts/sanitycheck

```diff
@@ -1138,17 +1151,28 @@ def main():
         if completed > 1:
             logger.info("%d Iteration:" % (completed))
             time.sleep(options.retry_interval)  # waiting for the system to settle down
-            suite.total_done = suite.total_tests - suite.total_failed
-            suite.total_failed = suite.total_errors
+            # FIXME
+            done1 = results.get_total() - results.get_failed()
+            failed1 = results.get_error()
```

Comment:
I see FIXME comment, but just in case: `done1` and `failed1` are used nowhere. What is the plan for them?